src/main/java/edu/cornell/kfs/fp/batch/service/impl/CuDisbursementVoucherDocumentGenerator.java

```diff
@@ -216,6 +216,13 @@ protected void addWarningMessageIfPayeeNameNotAccurate(String vendorPayeeName, S
     }
     
     private void populateAddressFields(DisbursementVoucherPaymentInformationXml paymentInfo, CuDisbursementVoucherDocument dvDocument) {
+        if (StringUtils.isNotBlank(paymentInfo.getPayeeAddressId()) && 
+                !StringUtils.equalsAnyIgnoreCase(paymentInfo.getPayeeTypeCode(), CUPdpConstants.PAYEE_TYPE_CODE_VENDOR)) {
```

Comment:
Thanks for noticing the typo, I shall fixed

I think the different if blocks makes it clearer.  The first block being validation code, the second block being logic setting payee details.  Let me know if you think otherwise

Reply:
Thanks for the feedback. I'm fine with your setup of splitting the code between the validation portion and the payee detail setup portion.